darts/timeseries.py

```diff
@@ -3607,6 +3612,57 @@ def _get_kwargs(transformation, forecasting_safe):
         if isinstance(transforms, dict):
             transforms = [transforms]
 
+        several_transforms = len(transforms) > 1
```

Comment:
you are defining this variable here but you are using it after 40 lines. 
I will put the variable definition above the line where you use it, or I will not use this variable at all.